app/ui/config.py

```diff
@@ -21,6 +21,10 @@ def up_to_date():
     return str(Config.get("post_setup")) == CurrentVersion.get()
 
 
+def get_current_and_expected_version():
+    return CurrentVersion.get(), Config.get("post_setup")
```

Comment:
Unused?

Reply:
currently yes. I added it b/c it was in v1 and I assume it will get used

Reply:
Fine to leave it in. Just didn't recall seeing it in the v1 PR and wanted to dbl check it was intentional.